docs/client-sdk/javascript/tutorials/start-messaging.md

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_label: Start messaging with XMTP
+sidebar_position: 1
+---
+
+# Start messaging with XMTP
+
+XMTP is an open protocol and network for secure web3 messaging. 
+
+Developers build with [XMTP client SDKs](/sdks-and-tools) to provide messaging in their apps. The XMTP messaging API client takes care of:
+
+- [Authentication](/docs/dev-concepts/account-signatures) using an **XMTP identity that you own and control**
+- [End-to-end encryption](/docs/dev-concepts/invitation-and-message-encryption) of **messages that you own and control**
+- Providing an **[interoperable inbox](/docs/dev-concepts/interoperable-inbox)** accessible across apps built with XMTP
+- Relaying messages to the **progressively decentralized** [XMTP network](/docs/dev-concepts/architectural-overview#network-layer)
+
+A great way to start learning how to build an app with XMTP is to use one!
+
+**To start messaging with XMTP:**
+
+1. **Pick an identity** you want to use to message with XMTP.
+
+    You can use an Ethereum address to start.
+    
+    If you have an [ENS name](https://ens.domains/) or [Lens handle](https://claim.lens.xyz/), you can use it instead. These human-readable web3 identities make it easier for you to share your address so others can message you.
+
+    Don’t have an Ethereum address? [Create a MetaMask wallet](https://metamask.io/) to get one.
+    
+2. **Pick an app.**
+
+   - [Converse](https://getconverse.app/) (iOS): Works with Ethereum addresses, ENS names, and Lens handles
+   - [XMTP Inbox](https://xmtp.chat/) (web): Works with Ethereum addresses and ENS names
+   - [Buttrfly](https://buttrfly.app/) (iOS): Works with Lens handles
+   - [Lenster](https://lenster.xyz/) (web): Works with Lens handles
+   - [Orb](https://orb.ac/) (ios and Android): Works with Lens handles
+
+3. **Send a message.**
+
+   To send a message to an address, it must have an XMTP identity already created on the network. Share this page with someone you want to message to help them get on the network. In the meantime, need someone to message? Try one of these addresses:
+   - hi.xmtp.eth: An XMTP Labs team member will send you a message back. The reply may not be instantaneous because this isn't a 🤖.
+   - ???
```

Comment:
Any volunteers? <3

Reply:
You can put me on there 😬 (hope I don't regret this later lol)

Dk if and who monitors hi bot